Library/OcCpuLib/OcCpuLib.c

```diff
@@ -853,17 +853,13 @@ ScanAmdProcessor (
   UINT64 CoreDivisorID;
   UINT64 Divisor;
   //
-  // Faking an Intel Core i5 Processor.
-  // This value is purely cosmetic, but it makes sense to fake something
-  // that is somewhat representative of the kind of Processor that's actually
-  // in the system
+  // Faking as unknown processor. This value is purely cosmetic.
   //
-  Cpu->AppleProcessorType = AppleProcessorTypeCorei5Type5;
+  Cpu->AppleProcessorType = AppleProcessorMajorUnknown;
```

Comment:
Why do you think unknown is better than e.g. i5? To be honest, I expected that Xeon would make more sense due to core count, but kind of still.

Reply:
using `AppleProcessorMajorUnknown` allows us to use real cpu name in About This Mac from sysctl `machdep.cpu.brand_string`
and setting the value in `/System/Library/PrivateFrameworks/AppleSystemInfo.framework/Versions/A/Resources/English.lproj`

>`<key>UnknownCPUKind</key>`
> `<string>Unknown</string>`
to
>`<key>UnknownCPUKind</key>`
> `<string>AMD FX 8320</string>`

Reply:
using `AppleProcessorMajorUnknown` allows us to use real cpu name in About This Mac from sysctl `machdep.cpu.brand_string`
and setting the value in `/System/Library/PrivateFrameworks/AppleSystemInfo.framework/Versions/A/Resources/English.lproj`

>`<key>UnknownCPUKind</key>`
> `<string>Unknown</string>`
to
>`<key>UnknownCPUKind</key>`
> `<string>AMD FX 8320</string>`

we use a script to do that automatically so the only change we need in OC is to set it to `AppleProcessorMajorUnknown` so that macos will use `<key>UnknownCPUKind</key>` value for About This Mac